package.json

```diff
@@ -2,50 +2,52 @@
   "name": "ember-resolver",
   "version": "4.3.0",
   "description": "The default modules based resolver for Ember CLI.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "Robert Jackson <me@rwjblue.com>",
   "directories": {
     "test": "tests"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ember-cli/ember-resolver.git"
+  },
   "scripts": {
     "build": "ember build",
     "start": "ember server",
     "test": "ember try:each"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ember-cli/ember-resolver.git"
-  },
-  "engines": {
-    "node": ">= 4"
+  "dependencies": {
+    "@glimmer/resolver": "^0.4.1",
+    "babel-plugin-debug-macros": "^0.1.10",
+    "broccoli-funnel": "^1.1.0",
+    "broccoli-merge-trees": "^2.0.0",
+    "ember-cli-babel": "^6.3.0",
+    "ember-cli-shims": "^1.1.0",
```

Comment:
Addons can use the new modules in their addon/ (and addon-test-support) tree as long as they themselves have ember-cli-babel@6.6.0 or higher as a dependency. 

Since an addon does not control the mechanism for transpilation of the app/ and test-support/ trees (these are compiled by the app's own preprocessor chain) an addon should _only_ reexport from their addon tree into app/ and from their addon-test-support tree in their test-support tree. 

To the specific point RE: the 'ember' module itself (which is not currently provided by ember-cli-babel@6.6.0), we just need to fix this and ensure that it is. We definitely do not want to add ember-cli-shims as a dependency to any addons.